docs/content/latest/benchmark/tpcc-ysql.md

```diff
@@ -92,6 +92,11 @@ $ ./tpccbenchmark --create=true --load=true --execute=true \
 ```
 {{< /tip>}}
 
+{{< note title="Note" >}}
```

Comment:
Let's add a widget like this: https://docs.yugabyte.com/latest/quick-start/explore-ysql/#macos

Each tab showing the instructions for a different number of warehouses. This note can be easily missed when trying to run the instructions.

Reply:
Done.